.github/workflows/test-invoke-pip-skip.yml

```diff
@@ -1,12 +1,13 @@
 name: Test invoke.py pip
 on:
   pull_request:
-    paths-ignore:
-      - 'pyproject.toml'
-      - 'invokeai/**'
-      - 'invokeai/backend/**'
-      - 'invokeai/configs/**'
-      - 'invokeai/frontend/web/dist/**'
+    paths:
+      - 'invokeai/frontend/web/**'
```

Comment:
why is `/web/**` now necesarry, shouldn't only `/web/dist/**` be related, since this includes the built website, while changes to files over `/web/dist` (`/web/**`) are not related unless they are built?

If not, the `/web/dist/**` can be removed (since /web/** would include this)

of course this comment applys to all other path-related triggers as well 🙈

Reply:
Ok, I somehow ignored the description of this PR 😅